app/mapping/playback/LeadTrusteeIndExtractor.scala

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package mapping.playback
+
+import java.time.LocalDate
+
+import com.google.inject.Inject
+import mapping.playback.PlaybackExtractionErrors.{FailedToExtractData, PlaybackExtractionError}
+import models.core.pages.{FullName, IndividualOrBusiness, InternationalAddress, UKAddress}
+import models.playback.http.DisplayTrustLeadTrusteeIndType
+import models.playback.{MetaData, UserAnswers}
+import models.registration.pages.PassportOrIdCardDetails
+import pages.register.trustees._
+import play.api.Logger
+
+import scala.util.{Failure, Success, Try}
+
+class LeadTrusteeIndExtractor @Inject() extends PlaybackExtractor[Option[DisplayTrustLeadTrusteeIndType]] {
+
+  import PlaybackAddressImplicits._
+
+  override def extract(answers: UserAnswers, data: Option[DisplayTrustLeadTrusteeIndType]): Either[PlaybackExtractionError, Try[UserAnswers]] =
+    {
+      data match {
+        case None => Right(Success(answers))
+        case leadTrustee =>
+
+          val updated = leadTrustee.foldLeft[Try[UserAnswers]](Success(answers)){
+            case (answers, leadTrustee) =>
+
+              answers
+                .flatMap(_.set(IsThisLeadTrusteePage(0), true))
+                .flatMap(_.set(TrusteeIndividualOrBusinessPage(0), IndividualOrBusiness.Individual))
+                .flatMap(_.set(TrusteesNamePage(0), FullName(leadTrustee.name.firstName, leadTrustee.name.middleName, leadTrustee.name.lastName)))
+                .flatMap(_.set(TrusteesDateOfBirthPage(0),
```

Comment:
Similar with address, can we have a common date of birth extractor that does the conversion?